dear-diary/src/utils/signin.ts

```diff
@@ -0,0 +1,7 @@
+export const generateRandomNickname = () => {
```

Comment:
change this file name to randomNameGenerator or something like that, I'll merge the changes for now fix this in a future PR